src/kontrol/prove.py

```diff
@@ -81,7 +81,10 @@ def foundry_prove(
     )
 
     if options.cse:
-        test_suite = collect_tests(foundry, options.tests, reinit=options.reinit, return_empty=True)
+        exact_match = options.config_type == ConfigType.SUMMARY_CONFIG
```

Comment:
Could you please add to the PR description the issue that this solves? If I'm remembering correctly, not having an exact match when running the underlying CSE functions would result in unrelated functions from various libraries being analysed?

Reply:
Yes yes, sure, I'm in the process of writing it :) thanks!

Reply:
Done!